enterprise/server/remote_execution/containers/podman/BUILD

```diff
@@ -53,3 +53,39 @@ go_test(
         "@org_golang_x_sync//errgroup",
     ],
 )
+
+# Repackages the podman-static distribution to make it more easily consumable
+# both by the container_layer rule and by tests:
+#
+# - Replaces the registry config with our custom one. This way, we don't have to
+#   override it separately.
+# - Strips the top-level podman-static directory. This is needed because the
+#   'tars' attribute of container_layer does not seem to support stripping
+#   components, currently.
```

Comment:
Nit: since we are unpacking this later to `/` root dir of the Executor container, it would be nice to have a comment here stating what's the expected file layout inside the output tarball.

Reply:
Done